### PR TITLE
Modify gradle properties to include data for publishing the plugin

### PR DESCRIPTION
**Pull request by frextrite**
Plugin is compiled with `Idea Community 192.5728.98`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -45,3 +45,6 @@ gradle-app.setting
 # Extra files
 **/main.iml
 **/test.iml
+
+# include Gradle run configurations
+!.idea/runConfigurations/
```
Comment (ksdme) [inline .gitignore]: Missing trailing new line.

File: build.gradle [hunk 2]
```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 group 'io.coala.jetbrains'
-version '0.0.1'
+version '1.0'
```
Comment (ksdme) [inline build.gradle]: I don't think we should bump it to 1. Maybe `0.1`? It needs some testing from other contributors as well.

File: src/main/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+    <id>io.coala.jetbrains</id>
     <name>coala Code Analysis</name>
     <vendor url="https://coala.io/">coala</vendor>
 
     <description><![CDATA[
-    <h1>coala: Language Independent Code Analysis</h1>
     coala provides a unified interface for linting and fixing all your code, regardless of the programming languages you use.
     ]]></description>
 
+    <change-notes><!-- --></change-notes>
```
Comment (ksdme) [inline src/main/resources/META-INF/plugin.xml]: `<!-- -->` needed?

**Review by ksdme — CHANGES_REQUESTED** [review]

**Review by ksdme — APPROVED** [review]
Good to go once my comments are addressed and build passes.

File: build.gradle (hunk 2 above)
Comment (frextrite) [reply build.gradle]: I was thinking the same, this is more like alpha/beta atm. I'll go with `0.1`

File: src/main/resources/META-INF/plugin.xml (hunk 3 above)
Comment (frextrite) [reply src/main/resources/META-INF/plugin.xml]: the `pathPluginXml` from `build.gradle` will over-write this

**Pull request merged**